src/AppInstallerRepositoryCore/SQLiteStatementBuilder.h

```diff
@@ -216,6 +216,7 @@ namespace AppInstaller::Repository::SQLite::Builder
         // Begin a filter clause on the given column.
         StatementBuilder& Where(std::string_view column);
         StatementBuilder& Where(const QualifiedColumn& column);
+        StatementBuilder& Or(const QualifiedColumn& column);

```

Comment:
Probably better next to the `And` (unless I'm misremembering and there is no `And`).